src/test-tree/createTests.ts

```diff
@@ -503,20 +510,21 @@ class FailedCase extends TestCase {
 }
 
 async function getCurrentRustFileMetadata(item: any): Promise<FileMetaData | undefined> {
-	const editor = vscode.window.activeTextEditor;
-	if (!editor) {
+	const workspaceFolders = vscode.workspace.workspaceFolders;
+
+	if (workspaceFolders === undefined || workspaceFolders?.length == 0) {
```

Comment:
```workspaceFolders?.length``` evaluates to `undefined` if `workspaceFolders` is `null` or `undefined`, right? I don't think you need the first condition.

Reply:
Sure, I can replace it with an invariant check that it `workspaceFolders` is of type array

Reply:
Wanted to suggest this instead:
```suggestion
	if (!workspaceFolders || workspaceFolders.length === 0) {
```
Because it's easier for me to read by avoiding the `?` operator, but also because it uses `===` for comparison.

Reply:
Agreed, that's much more readable